### PR TITLE
feat/update_demo_app_revenue_tracking_with_adjust_sdk

### PR DESCRIPTION
**Pull request by Billy-Hu**

### CODE REVIEW

File: AppLovin MAX Demo App - Java/app/src/main/java/com/applovin/enterprise/apps/demoapp/GlobalApplication.java [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+    public void onCreate()
+    {
+        super.onCreate();
+
+        // Initialize the AppLovin SDK
+        AppLovinSdk.getInstance( this ).setMediationProvider( AppLovinMediationProvider.MAX );
+        AppLovinSdk.getInstance( this ).initializeSdk( config -> {
+            // AppLovin SDK is initialized, start loading ads now or later if ad gate is reached
+            // set up Adjust SDK
+            String appToken = "{YourAppToken}";
```
Comment (thomasmso) [inline AppLovin MAX Demo App - Java/app/src/main/java/com/applovin/enterprise/apps/demoapp/GlobalApplication.java:28]:
```suggestion
        AppLovinSdk.getInstance( this ).initializeSdk( config -> {
        
            // AppLovin SDK is initialized, start loading ads now or later if ad gate is reached
            
            // Set up Adjust SDK
            String appToken = "{YourAppToken}";
```

File: AppLovin MAX Demo App - Kotlin/app/src/main/AndroidManifest.xml [hunk 2]
```diff
@@ -12,7 +13,7 @@
 
         <meta-data
             android:name="applovin.sdk.key"
-            android:value="SDK_KEY_HERE" />
+            android:value="hBDh6tzZrp-fWye63N4nhbgw8umnTzD99QsGIpq8bpo7lRDppHZVuEQ17Bpa80lIRaTlONt-Af6v5JiubGOUVp" />
```
Comment (thomasmso) [inline AppLovin MAX Demo App - Kotlin/app/src/main/AndroidManifest.xml:16]: No need to check our SDK key

File: AppLovin MAX Demo App - Kotlin/app/src/main/java/com/applovin/enterprise/apps/demoapp/GlobalApplication.kt [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+
+    override fun onCreate() {
+        super.onCreate()
+
+        // Initialize the AppLovin SDK
+        AppLovinSdk.getInstance(this).mediationProvider = AppLovinMediationProvider.MAX
+        AppLovinSdk.getInstance(this).initializeSdk {
+            // AppLovin SDK is initialized, start loading ads now or later if ad gate is reached
+            // set up Adjust SDK
+            val appToken = "{YourAppToken}"
```
Comment (thomasmso) [inline AppLovin MAX Demo App - Kotlin/app/src/main/java/com/applovin/enterprise/apps/demoapp/GlobalApplication.kt:22]:
```suggestion
        AppLovinSdk.getInstance(this).initializeSdk {
            
            // AppLovin SDK is initialized, start loading ads now or later if ad gate is reached

            // Set up Adjust SDK
            val appToken = "{YourAppToken}"
```

File: AppLovin MAX Demo App - Kotlin/app/src/main/java/com/applovin/enterprise/apps/demoapp/GlobalApplication.kt [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+            val environment = AdjustConfig.ENVIRONMENT_SANDBOX
+            val config = AdjustConfig(this, appToken, environment)
+            Adjust.onCreate(config)
+
+            registerActivityLifecycleCallbacks(AdjustLifecycleCallbacks())
+        }
+    }
+
+    private class AdjustLifecycleCallbacks : ActivityLifecycleCallbacks {
+        override fun onActivityCreated(p0: Activity, p1: Bundle?) {}
```
Comment (thomasmso) [inline AppLovin MAX Demo App - Kotlin/app/src/main/java/com/applovin/enterprise/apps/demoapp/GlobalApplication.kt:32]: We should give these parameters better names, like we did in Java.

**Review by thomasmso — CHANGES_REQUESTED** [review]

**Comment by Billy-Hu** [comment]
Changes made

File: AppLovin MAX Demo App - Java/app/src/main/java/com/applovin/enterprise/apps/demoapp/GlobalApplication.java [hunk 5]
```diff
@@ -0,0 +1,70 @@
...
+        AppLovinSdk.getInstance( this ).setMediationProvider( AppLovinMediationProvider.MAX );
+        AppLovinSdk.getInstance( this ).initializeSdk( config -> {
+            // AppLovin SDK is initialized, start loading ads now or later if ad gate is reached
+
+            // set up Adjust SDK
+            String appToken = "{YourAppToken}";
+            String environment = AdjustConfig.ENVIRONMENT_SANDBOX;
+            AdjustConfig adjustConfig = new AdjustConfig( getApplicationContext(), appToken, environment );
+            adjustConfig.setLogLevel( LogLevel.VERBOSE );
+            Adjust.onCreate( adjustConfig );
```
Comment (thomasmso) [inline AppLovin MAX Demo App - Java/app/src/main/java/com/applovin/enterprise/apps/demoapp/GlobalApplication.java:34]: Let's inline params and keep comment consistent with iOS, and no need for verbose log setting
```suggestion
            // Initialize Adjust SDK
            AdjustConfig adjustConfig = new AdjustConfig( getApplicationContext(), "{YourAppToken}", AdjustConfig.ENVIRONMENT_SANDBOX );
            Adjust.onCreate( adjustConfig );
```

File: AppLovin MAX Demo App - Kotlin/app/src/main/java/com/applovin/enterprise/apps/demoapp/GlobalApplication.kt [hunk 6]
```diff
@@ -0,0 +1,53 @@
...
+        AppLovinSdk.getInstance(this).mediationProvider = AppLovinMediationProvider.MAX
+        AppLovinSdk.getInstance(this).initializeSdk {
+            // AppLovin SDK is initialized, start loading ads now or later if ad gate is reached
+
+            // set up Adjust SDK
+            val appToken = "{YourAppToken}"
+            val environment = AdjustConfig.ENVIRONMENT_SANDBOX
+            val config = AdjustConfig(this, appToken, environment)
+            config.setLogLevel(LogLevel.VERBOSE)
+            Adjust.onCreate(config)
```
Comment (thomasmso) [inline AppLovin MAX Demo App - Kotlin/app/src/main/java/com/applovin/enterprise/apps/demoapp/GlobalApplication.kt:28]:
```suggestion
            // Initialize Adjust SDK
            val config = AdjustConfig(this, "{YourAppToken}", AdjustConfig.ENVIRONMENT_SANDBOX)
            Adjust.onCreate(config)
```

**Comment by Billy-Hu** [comment]
Changes made

File: AppLovin MAX Demo App - Java/app/src/main/java/com/applovin/enterprise/apps/demoapp/GlobalApplication.java [hunk 7]
```diff
@@ -0,0 +1,67 @@
...
+        super.onCreate();
+
+        // Initialize the AppLovin SDK
+        AppLovinSdk.getInstance( this ).setMediationProvider( AppLovinMediationProvider.MAX );
+        AppLovinSdk.getInstance( this ).initializeSdk( config -> {
+            // AppLovin SDK is initialized, start loading ads now or later if ad gate is reached
+
+            // set up Adjust SDK
+            AdjustConfig adjustConfig = new AdjustConfig( getApplicationContext(), "{YourAppToken}", AdjustConfig.ENVIRONMENT_SANDBOX );
+            Adjust.onCreate( adjustConfig );
```
Comment (thomasmso) [inline AppLovin MAX Demo App - Java/app/src/main/java/com/applovin/enterprise/apps/demoapp/GlobalApplication.java:31]:
```suggestion
            // Initialize Adjust SDK
            AdjustConfig adjustConfig = new AdjustConfig( getApplicationContext(), "{YourAppToken}", AdjustConfig.ENVIRONMENT_SANDBOX );
            Adjust.onCreate( adjustConfig );
```

File: AppLovin MAX Demo App - Kotlin/app/src/main/java/com/applovin/enterprise/apps/demoapp/GlobalApplication.kt [hunk 8]
```diff
@@ -0,0 +1,50 @@
...
+
+    override fun onCreate() {
+        super.onCreate()
+
+        // Initialize the AppLovin SDK
+        AppLovinSdk.getInstance(this).mediationProvider = AppLovinMediationProvider.MAX
+        AppLovinSdk.getInstance(this).initializeSdk {
+            // AppLovin SDK is initialized, start loading ads now or later if ad gate is reached
+
+            // set up Adjust SDK
```
Comment (thomasmso) [inline AppLovin MAX Demo App - Kotlin/app/src/main/java/com/applovin/enterprise/apps/demoapp/GlobalApplication.kt:23]:
```suggestion
            // Initialize Adjust SDK
```

**Review by thomasmso — APPROVED** [review]

**Review by thomasmso — APPROVED** [review]

**Pull request merged**